library/scripts/.util/tools.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We'll want to include changes to the workflow to update this file so it stays up-to-date: https://github.com/paketo-buildpacks/github-config/blob/f3bcea469f69d15417fbd75a600abbc6d9f4d91f/.github/workflows/update-tools.yml#L52-L99

Reply:
Hmm.. I hadn't seen this file before. Not sure what the action is to make sure it is updated.

Reply:
Oh I see. For now I'll actually remove `tools.json` since `jam` and `pack` (etc) aren't being used by `libdependency` and I'm not sure that they are needed for some general library integration testing. Will push that in a minute.

Reply:
Resolving since library no longer has `tools.json`.